application/src/main/resources/thingsboard.yml

```diff
@@ -1644,7 +1644,7 @@ queue:
       # Kafka properties for Notifications topics
       notifications: "${TB_QUEUE_KAFKA_NOTIFICATIONS_TOPIC_PROPERTIES:retention.ms:604800000;segment.bytes:52428800;retention.bytes:1048576000;partitions:1;min.insync.replicas:1}"
       # Kafka properties for JS Executor topics
-      js-executor: "${TB_QUEUE_KAFKA_JE_TOPIC_PROPERTIES:retention.ms:604800000;segment.bytes:52428800;retention.bytes:104857600;partitions:100;min.insync.replicas:1}"
+      js-executor: "${TB_QUEUE_KAFKA_JE_TOPIC_PROPERTIES:retention.ms:86400000;segment.bytes:52428800;retention.bytes:104857600;partitions:30;min.insync.replicas:1}"
```

Comment:
Please explain the change of partition count from 10 to 30?

Reply:
Given that 1 executor utilizes 1 vCPU.
I did not see more than 30 executors.
Most of deployments are monolith with moderate js load.
Not affected already existed deployments (topic properties used are for new topics only)
30 chosen as a perfect distribution for  replicas = 1,2,3,5,6,10,15,30